app/Http/Controllers/Assets/AssetFilesController.php

```diff
@@ -110,15 +110,15 @@ public function destroy($assetId = null, $fileId = null)
     {
         $asset = Asset::find($assetId);
         $this->authorize('update', $asset);
-        $rel_path = 'storage/private_uploads/assets';
```

Comment:
Actually, why did you remove the `storage` from the path here? `private_uploads` lives in the `storage` directory.

Reply:
Because looking higher up in the file it appears the Storage service is relative to that storage directory (See: https://github.com/misilot/snipe-it/blob/89de8c2213cfad83a40b3959ace9f9563d654979/app/Http/Controllers/Assets/AssetFilesController.php#L41).

I think that is partially why it wasn't being removed correctly.